barcodes_generator_abstract/models/barcode_rule.py

```diff
@@ -153,4 +150,4 @@ def _clear_cache(self, vals):
         """It clears the caches if certain vals are updated."""
         fields = ("generate_model", "generate_automate")
         if any(k in vals for k in fields):
-            self.invalidate_cache(fields)
+            self.invalidate_model(fields)
```

Comment:
Question: have you tested without this _clear_cache() function ?

Reply:
Hi. 
I don't know why this part has been introduced.
I removed this part and the test still works. ;-)
It has been intro by @lasley in V13 https://github.com/OCA/stock-logistics-barcode/commit/51d9ef002e86d86e1b0226d118e1e0cbd6b0f134

No idea what to do. I remove ?